install_config/cfme/container_provider.adoc

```diff
@@ -0,0 +1,180 @@
+* `hostname` - This is the host name that points to the cluster API. Each container provider must have a unique host name.
+* `name` - This is the name of the cluster to be displayed in the {cloudforms} server container providers overview page. This must be unique.
+--
++
+[TIP]
+====
+To obtain the `auth_key` bearer token from your clusters:
+
+----
+$ oc serviceaccounts get-token -n management-infra management-admin
+----
+====
+
+.. The following parameters may be optionally configured:
++
+--
+* `port` - Update this key if your container platform cluster runs the API on a port other than `8443`.
+* `endpoint` - You may enable SSL verification (`verify_ssl`) or change the validation setting to `ssl-with-validation`. Support for custom trusted CA certificates is not currently available.
+--
+
+==== Example
+
+As an example, consider the following scenario:
+
+- You copied the *_container_providers.yml_* file to *_/tmp/cp.yml_*.
+- You want to add two OpenShift Container Platform clusters.
+- Your {cloudforms} server runs on `mgmt.example.com`
+
+For this scenario, you would customize *_/tmp/cp.yml_* as follows:
+
+[source,yaml]
+----
+container_providers:
+  - connection_configurations:
+      - authentication: {auth_key: "management-token-for-this-cluster", authtype: bearer, type: AuthToken}
+        endpoint: {role: default, security_protocol: ssl-without-validation, verify_ssl: 0}
+    hostname: "ocp-prod.example.com"
+    name: OCP Production
+    port: 8443
+    type: "ManageIQ::Providers::Openshift::ContainerManager"
+  - connection_configurations:
+      - authentication: {auth_key: "management-token-for-this-cluster", authtype: bearer, type: AuthToken}
+        endpoint: {role: default, security_protocol: ssl-without-validation, verify_ssl: 0}
+    hostname: "ocp-test.example.com"
+    name: OCP Testing
+    port: 8443
+    type: "ManageIQ::Providers::Openshift::ContainerManager"
+management_server:
+  hostname: "mgmt.example.com"
+  user: admin
+  password: mypassword
+----
+
+=== Running the Playbook
+
+To run the multiple-providers integration script, you must provide the path to
+the container providers configuration file as an `EXTRA_VARS` parameter to the
+`ansible-playbook` command. Use the `-e` (or `--extra-vars`) parameter to set
+`container_providers_config` to the configuration file path:
+
+----
+# ansible-playbook -v [-i /path/to/inventory] \
+    -e container_providers_config=/tmp/cp.yml \
+ifdef::openshift-origin[]
+    playbooks/byo/openshift-management/add_many_container_providers.yml
+endif::[]
+ifdef::openshift-enterprise[]
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-management/add_many_container_providers.yml
+endif::[]
+----
+
+After the playbook completes, you should find two new container providers in
+your {cloudforms} service. Navigate to the `Compute → Containers → Providers`
+page to see an overview.
```

Comment:
Both of the playbooks, *automatic addition* and *multiple addition*, require one extra step after running. The new providers have to be refreshed in MIQ/CFME to get all the latest data about the provider (OCP) and the containers being managed.

Here's the docs for how to do that

http://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#_refreshing_multiple_management_systems

Looks like you have to manually go to each provider and click the refresh button for each.

*again*: This applies to the automatic addition playbook *as well as* the multi-adder playbook